README.md

```diff
@@ -3,11 +3,15 @@ Docker image of ROS (Robot Operating System) on Alpine Linux
 
 ## News
 
+### January 2024
+`noetic-3.14` (Alpine 3.14 EOL on May 2023) are dropped. These corresponding images are still available but no longer updated.
+It is highly recommended to update to `noetic-3.17`.
+
 ### May 2023
 ROS2 humble is now available for Alpine 3.17 as `humble-3.17`.
 
 ### April 2023
-`melodic` (Melodic EOL on May 2023) and `noetic-3.11` (Alpine 3.11 EOL on May 2020) are dropped.
+`kinetic` (Kinetic EOL on May 2021), `melodic` (Melodic EOL on June 2023) and `noetic-3.11` (Alpine 3.11 EOL on November 2021) are dropped.
```

Comment:
We dropped kinetic build on 2021
https://github.com/alpine-ros/alpine-ros/pull/18

Reply:
I just reverted the changes.